src/WebJobs.Script.WebHost/Helpers/SimpleWebTokenHelper.cs

```diff
@@ -47,6 +45,38 @@ private static string Encrypt(string value)
             }
         }
 
+        public static string Decrypt(byte[] encryptionKey, string value)
```

Comment:
Please add an Encrypt/Decrypt test for this, if one doesn't already exist in your branch.